modules/schedule/main.tf

```diff
@@ -2,9 +2,9 @@ resource "opsgenie_schedule" "default" {
   count = module.this.enabled ? 1 : 0
 
   name          = var.schedule.name
-  description   = try(var.schedule.description, var.schedule.name)
+  description   = try(var.schedule.description, null)
   owner_team_id = try(var.schedule.owner_team_id, null)
 
-  enabled  = try(var.schedule.enabled, true)
+  enabled  = try(var.schedule.enabled, null) # this is the state of the schedule, NOT module.this.enabled
```

Comment:
can it be set to `null`, ot it always requires `true` or `false`?

Reply:
This can be set to `null` and if it's set to null on the raw resource, the resource will omit the input and the value of `enabled` will be set to whatever the provider (or API) has for a default value. Presumably a value of `true` if it's omitted.

The documentation does not show what the default value is but it does show that this input is optional.

https://registry.terraform.io/providers/opsgenie/opsgenie/latest/docs/resources/schedule#enabled